src/components/policy/policy_regular/src/cache_manager.cc

```diff
@@ -564,16 +564,22 @@ bool CacheManager::SetCountersPassedForSuccessfulUpdate(
 int CacheManager::DaysBeforeExchange(int current) {
   LOG4CXX_AUTO_TRACE(logger_);
   CACHE_MANAGER_CHECK(0);
+
+  const rpc::Optional<rpc::Integer<uint16_t, 0, 65535> > days_after_epoch =
+      (pt_->policy_table.module_meta->pt_exchanged_x_days_after_epoch);
+
```

Comment:
@indlu , @tkireva should use the same type as in PT. So it is ok that is is Optional from POlicies.